### ISSUE TITLE
DomainNet won’t run; OT-cost mismatch (mbtc.py vs duct.py); CDDB forgetting gap; accuracy divergence

### ISSUE THREAD

**Issue by mnJ00** [Estrella-fugaz/CVPR25-DUCT#7]
Title: DomainNet won’t run; OT-cost mismatch (mbtc.py vs duct.py); CDDB forgetting gap; accuracy divergence

Hi authors—thanks for releasing the code. I’m trying to reproduce your results and ran into several issues:

1. DomainNet doesn’t run

              -I cannot get DomainNet to train with the current repo.
              
              -Request: Could you share a working config/script (and any dataset preprocessing instructions) for DomainNet so it runs end-to-end?
              
2. OT cost appears in your logs (mbtc.py) but not when running duct.py

              -In the log you shared, the training references mbtc.py and prints “The OT cost …”.
              
              -When I run the provided duct.py, there is no OT-cost computation or printout.
              
              -Question: Is the log from an older/different code path (e.g., mbtc.py) than this repo? If so, please provide the exact script/commit that produced those logs, or clarify how to enable the OT step in duct.py.

3. CDDB “Forgetting” discrepancy

            - Your CDDB log shows Forgetting (after last task): 6.145, but the paper reports 0.12.
            
            - Question: Are these computed with different definitions, scales, or evaluation protocols (e.g., per-task averaging, normalization, class-increment schedule, or domain order)? Please clarify the exact forgetting formula and evaluation script you used for the paper, and share the code that reproduces 0.12.

4. Large accuracy gap vs. your logs

            - Running the repo as provided yields accuracies that differ substantially from the numbers in your shared logs.
            
            - Request: Please share the exact commit hash, config files, seeds, domain/class order, and any non-default hyperparameters or checkpoints used to generate your logs.

If helpful, I can attach my full console logs and configs for each run. Sharing the exact scripts/configs (and any missing components) used for DomainNet/CDDB would make reproduction much easier. Thanks!

<img width="429" height="275" alt="Image" src="https://github.com/user-attachments/assets/169a16eb-0278-4e20-8a1e-f1ad16fe8a01" />

**Comment by Estrella-fugaz** [comment]
1. If the code doesn’t run, please first check whether the dataset is complete and correctly prepared. Also, since no error message was provided, it’s difficult to determine the cause.

2. You can add additional logging in the code yourself — extra debug logs were removed in the released version.

3. As stated in the paper, all reported results are averaged over five task orders. Please double-check this detail.

4. The code and the paper already provide the corresponding configurations and task sequences.
